news/_posts/2022-01-13-new-theme-announce.md

```diff
@@ -0,0 +1,71 @@
+-   Use lazy loading.
+-   Choose, and use some CSS class naming strategy consistently, like BEM. This is more maintainable and easier to read.
+-   Use JSON-LD instead of microdata attributes.
+-   Add as many hooks as possible to extend the base theme.
+-   Separate the PrestaShop styling from the Bootstrap raw styling.
+-   Reduce the number of dependencies.
+-   Stop using jQuery in the theme. This would be the first step because modules use it anyway, so we can't remove it.
+-   Most of the time, JavaScript completely replaces some big parts of the content. Instead, it should update only a small part of it when needed.
+-   Add as many events as possible in the JavaScript part so modules and themes can use them.
+-   Use as many Bootstrap components as possible.
+
+It was pretty impossible to fix all these issues on the classic theme without breaking the whole ecosystem of modules and community themes. We decided that it was time for a change and a new theme. We can't change everything, but we can significantly improve the default PrestaShop theme.
+
+## How can I try it?
+
+Clone it from the theme repository, run `npm install`, `npm run build`, activate the theme and off you go!
+
+## Good base for theme developers
+
+In the new theme, we separated the CSS to two packs. First one is the functionality part, the second is purely for styling.
+
+Simply commenting out `@import "custom/_custom";` in theme.scss file will get you basically a pure Boostrap 5 theme base. You can get working immediately and not waste time by cleaning things you don't need.
+
+## When will it be released?
+
+Because of the complexity of this task, we decided to avoid writing about the release date. This means that depending on the time every people involved can work on it, the first version of the theme can be released in PrestaShop v8, v9, or later.
+
+We hope to release an Alpha version of the new theme at the same time as the v8 release, but it could be after, even inside a minor version.
+
+This release date doesn't only depend on us, but it also depends on many contributors working hard to make this happen, so we would like to thank every contributor you can find [here](https://github.com/PrestaShop/theme-refacto/graphs/contributors) and those involved in any issues and reviews! Keep the great work. It's crucial for us. This theme is also your theme, don't hesitate to contribute and join the adventure :-)
+
+## How does the new theme look?
+
+You can find the mockups of the theme [here](https://www.figma.com/file/LfVl5leeSKcVUhSaYwhbtM/New-Theme), and [the list of every issue](https://github.com/PrestaShop/theme-refacto/issues) opened.
+
+## What about the Classic theme?
+
+Because a new theme means more maintainer works, we decided that the Classic theme will be out of support in the future. We didn't determine the date because the new theme release date is unknown.
```

Comment:
This part is going to worry some people. I think we must reassure prestashop community that **Classic Theme will work on both 1.7 and 8.0** and that we will deprecate Classic Theme in a timely manner in order to make it smooth for people who use it to transition.

Reply:
Isn't it the case already?

Reply:
Maybe we can make it even more clear :)

```
Building a new front office theme means to maintain it. In the past it was tried to support and maintain two front office themes simultaneously but the amount of work was too high for the maintainer team size.

We consequently decided to prepare the end-of-life of Classic theme that was PrestaShop 1.7 default front office theme. We do not have a fixed end-of-life date yet as the work on the new theme has no release date either.

As we get closer to the completion of this project, we have a better understanding about when we can release this new theme. As soon as we can, we will announce a date
- at this date, the new theme version 1.0.0 will be published
- at this date, the Classic theme will become [deprecated](https://en.wikipedia.org/wiki/Deprecation)

Deprecated does not mean it is not maintained anymore. There will be a period of time where Classic theme will be supported to a certain extent in order to allow theme developers who rely on it to start using the new theme. 

Until this date, Classic theme remains the default PrestaShop theme and is fully supported.
```

Is this acted yet? When we publish the new theme, we deprecate the old one or we stop supporting it? or do we have to discuss about a deprecation period where we maintain it?

Reply:
I'm not even sure if this is acted, but I think so